### ISSUE TITLE
Port Request: qt(kde) theme

### ISSUE THREAD

**Issue by RAVENz46** [catppuccin/nix#135]
https://github.com/catppuccin/kde

**Comment by IQubic** [comment]
It would be nice if we could get a QT theming solution that works on all WMs and DEs, but that might harder to implement.

**Comment by pluiedev** [comment]
> It would be nice if we could get a QT theming solution that works on all WMs and DEs, but that might harder to implement.

The thing is that such a thing is just not really possible due to the fun existence of Qt platforms. When I used Hyprland I used qt5ct/qt6ct, but switching back to KDE made my themes all break horrendously since apparently KDE does their own thing... I do have both qt{5,6}ct and KDE settings in my flake but I'm not sure what's the best approach to upstream the latter, since it depends on modifying `kdeglobals` (scary!)

**Comment by RAVENz46** [comment]
@pluiedev Yes, qt option was broken after releasing kde6. This problem also mentioned nix-community/home-manager#5098 .
I think it is caused by mismatching between libsForQt5, qt6Packages(deprecated but still used in [qt modules](https://github.com/nix-community/home-manager/blob/master/modules/misc/qt.nix)) and kde6Packages.
We have to wait until fixing it.

**Comment by madfanat** [comment]
> qt option was broken after releasing kde6

Well, I face a similar problem on Hyprland: no Catppuccin theme in kvantum and thus no theming at all. Any thoughts on how to fix this temporarily?

**Comment by RAVENz46** [comment]
https://github.com/nix-community/home-manager/pull/6493 got merged.
I think it's ready to go.
